### PR TITLE
 Updated README.md file

### PR DESCRIPTION
**Pull request by nextwebb**
Updated the README markdown file with Pay Tax App details.

### CODE REVIEW

**Review by Ebugo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,55 +1,171 @@
...
+-  Easily get tax receipts. 
 
 ## Usage
- How would someone use what you have built, include URLs to the deployed app, service e.t.c when you have it setup
+### API guide for consuming this back-end layer. 
+
+**Individual businesses and Companies registration.**
+Upon registration the Tax payer gets returned a Tax Payer ID by email and json response.
+
+`POST`  https://paytax-app.herokuapp.com/api/v1/users/register
```
Comment (triple0t) [inline README.md:25]: is this supposed to be _**users**/register_ or _**user**/register_?
What happens when we send a GET request to this endpoint?
Can you add a fallback check for all requests for POSTs

File: README.md [hunk 2]
```diff
@@ -1,55 +1,171 @@
...
+	"password":"password123456"
+}`
+
+****
+
+**Business data for Authenticated users.**
+
+Returns all company and individual biodata.
+
+`GET`  https://paytax-app.herokuapp.com/api/v1/use/biodata/:taxID
```
Comment (triple0t) [inline README.md:76]: is this accurate: _api/v1/**use**/biodata/:taxID_ , why not add it under _api/v1/user/_

**Review by triple0t — APPROVED** [review]
Kindly find comments
Also, fix Codacy/PR Quality Review Issues
Well done 👏🏿

File: README.md (hunk 2 above)
Comment (nextwebb) [reply README.md:76]: :( its a typo. it's api/v1/user/

File: README.md (hunk 1 above)
Comment (nextwebb) [reply README.md:25]: OKay. i'll work on a response feedback

**Pull request merged**